SConstruct

```diff
@@ -704,6 +704,7 @@ else:
 
     env.Append(ROC_TARGETS=[
         'target_libuv',
+        'target_openssl',
```

Comment:
We should add new option `--disable-openssl` and add this target only if the option is not set.

Reply:
done